src/main/java/in/twizmwaz/cardinal/module/modules/kit/KitBuilder.java

```diff
@@ -47,27 +47,12 @@ public static Kit getKit(Element element, Document document, boolean proceed) {
             armors.addAll(element.getChildren("leggings"));
             armors.addAll(element.getChildren("boots"));
             for (Element piece : armors) {
-                ItemStack itemStack = new ItemStack(Material.matchMaterial(piece.getText()), 1);
-                if (piece.getAttributeValue("damage") != null) {
-                    itemStack.setDurability(Short.parseShort(piece.getAttributeValue("damage")));
-                }
+                ItemStack itemStack = Parser.getItem(piece);
                 if (itemStack.getItemMeta() instanceof LeatherArmorMeta && piece.getAttributeValue("color") != null) {
                     LeatherArmorMeta meta = (LeatherArmorMeta) itemStack.getItemMeta();
                     meta.setColor(MiscUtil.convertHexToRGB(piece.getAttributeValue("color")));
                     itemStack.setItemMeta(meta);
                 }
-                try {
```

Comment:
We can't just delete this, older maps may still use this system to enchant items in their XML.